src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/DxAddressValidator.java

```diff
@@ -3,7 +3,7 @@
 import org.springframework.stereotype.Service;
 
 @Service
-public class DxAddressValidator implements OrganisationRequestValidator {
+public class DxAddressValidator implements IRequestValidator {

```

Comment:
This class is having empty validate method. delete this class if not doing anything.